plugins/modules/discovery_workflow_manager.py

```diff
@@ -64,65 +65,137 @@
         description: Start index for the header in fetching SNMP v2 credentials
         type: int
         default: 1
-      enable_password_list:
-        description: List of enable passwords for the CLI crfedentials
-        type: list
-        elements: str
       records_to_return:
         description: Number of records to return for the header in fetching global v2 credentials
         type: int
         default: 100
-      http_read_credential:
-        description: HTTP read credentials for hosting a device
+      discovery_specific_credentials:
+        description: Credentials specifically created by the user while performing discovery.
```

Comment:
Credentials specifically created by the user for performing device discovery.

Reply:
Addressed